pkgs/by-name/fa/fastfetch/package.nix

```diff
@@ -162,8 +171,12 @@ stdenv'.mkDerivation (finalAttrs: {
   meta = {
     description = "Like neofetch, but much faster because written in C";
     homepage = "https://github.com/fastfetch-cli/fastfetch";
+    changelog = "https://github.com/fastfetch-cli/fastfetch/releases/tag/${finalAttrs.version}";
```

Comment:
This change should be in a separate commit

Reply:
Still in the nixfmt commit but not a serious issue so resolving

Reply:
yeah it's not really a big deal.